Tensorflow/02_BuildImage.ipynb

```diff
@@ -116,12 +249,15 @@
     "import logging\n",
     "import json\n",
     "import driver\n",
+    "from .apisecurity import require_appkey\n",
+    "from .applicationinsights import *\n",
```

Comment:
So we are still installing application insights but not using it? I think this is a redundant dependency

Reply:
No, not installing anything. Just showing all they need to do to include it IF the file wasn't all comments.

Reply:
Yea, spotted that.